digits/templates/job.html

```diff
@@ -235,6 +235,53 @@ <h4 class="panel-title">
         </div>
         <div class="gpu-utilization-info" style="display:none;"></div>
 
+
+        <!-- Related jobs -->
+        {% if related_jobs is not none and related_jobs|length > 0 %}
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h4 class="panel-title">
+                    Related jobs
+                </h4>
+            </div>
+            <div class="panel-collapse">
+                <div class="panel-body">
+                    <div class="panel-group">
+                        <!-- Related model jobs -->
+                        {% for r_job in related_jobs %}
+                        {% if prev_job_model != r_job.job_type() %}
```

Comment:
it might have been more explicit to call this `prev_job_type` but I don't think it's a big deal